data/json/monsters/feral_humans.json

```
@@ -74,5 +74,126 @@
     "melee_dice": 3,
     "melee_dice_sides": 8,
     "death_drops": "feral_humans_death_drops_axe"
+  },
+  {
+    "id": "mon_feral_scientist_scalpel",
+    "type": "MONSTER",
+    "name": { "str": "mad scientist" },
+    "description": "A researcher who has stared too long into the abyss and now shambles about muttering nonsense under their breath. For some reason, the zombies don't seem to mind the fact that this person is obviously still alive. They're clutching a tiny razor-sharp blade in one shaky hand.",
+    "default_faction": "zombie",
```

```suggestion
    "default_faction": "science",
```
Recognized (as with their zombies) by the lab's security systems.